### DISCUSSION TITLE
Does anybody use the SMS feature?

### DISCUSSION THREAD

**Discussion by coccoinomane** [coccoinomane/crabada.py#42] [Ideas]
If not, I will remove it: now that we have Telegram I do not see much use for SMS.

**Comment by loayei** [comment]
Personally Telegram is way cleaner

**Comment by mertf** [comment]
I doubt any of us using the SMS over Telegram.

**Comment by coccoinomane** [comment]
Thanks guys for your feedback.
I will keep the poll open for a couple of days, after which I will remove the SMS feature.

**Comment by nikohd12** [comment]
Telegram is the way.
